### DISCUSSION TITLE
Improve App Icon

### DISCUSSION THREAD

**Discussion by sayyid5416** [huynhsontung/Screenbox#334] [Ideas]
This app feels modern in all of its aspects and looks professionally made BUT its icon doesn't looks so great _(in my subjective opinion)_. I believe there is room for improvement in its icon.
![image](https://github.com/huynhsontung/Screenbox/assets/78584556/92c8dfbd-7ec5-44c5-aa43-6834e85c140c)

**Comment by huynhsontung** [comment]
This is an open invitation for a designer to redesign the Screenbox logo 🥲

**Comment by sayyid5416** [comment]
I would be happy to design it. Tell me if you have any suggestions, like color, shape or some inspiration.

**Comment by huynhsontung** [comment]
Thank you for the offer. There are only two requirements for the Screenbox icon:
- The icon must relate to or suggest a media player.
- Somewhat follows the "Fluent icon style" that can be found in the Microsoft Office suite and Windows 11 native app icons.

Please reach out to me on Discord @highpine if you want to keep in touch.

**Comment by Mitia99** [comment]
> Thank you for the offer. There are only two requirements for the Screenbox icon:
> 
> * The icon must relate to or suggest a media player.
> * Somewhat follows the "Fluent icon style" that can be found in the Microsoft Office suite and Windows 11 native app icons.
> 
> Please reach out to me on Discord @highpine if you want to keep in touch.

I created this icon which give the Screenbox vibe (Box + Play icon) IMO.
I'm not that familiar with Fluent icon style design guidelines but It kinda match with Windows 11 apps icons.
Maybe you can revisit it with some gradient and make it more visible

I'm not a designer btw.

Thanks for considering this,
![box_dismiss_filled_icon_202504](https://github.com/huynhsontung/Screenbox/assets/56766778/6500b562-d7d4-4e23-beaa-2a293760d2b9)
![Screenshot 2023-11-21 082454](https://github.com/huynhsontung/Screenbox/assets/56766778/520917be-5424-4a31-935e-16df491c3493)

**Comment by sayyid5416** [comment]
@Mitia99 Icon looks good, except for the location of play icon. Its current location suggestion like it's from some sort of notification, and not the part of the icon, but I like the design of Cube.

**Comment by sayyid5416** [comment]
I also designed these two icons and sent it on discord. Lets see if owner of this app likes it.
![icon](https://github.com/huynhsontung/Screenbox/assets/78584556/db10350c-9e65-4841-83e2-94abb079d0ea)

There is no app which matches this icon and it matches perfectly with windows 11 _(specially the blue one)_

**Comment by maisondasilva** [comment]
I'm not a design person, but I found some cool versions that could serve as inspiration!

I think Play Button Circled is really good 
https://icons8.com/icon/JbiYfhbzVpRi/play-button-circled

What do you think about the icon @huynhsontung ?

https://icons8.com/icon/22skWjUReayF/video

https://icons8.com/icon/44057/circled-play

https://icons8.com/icon/63671/circled-play

How to use our graphics for free
Here's the deal. We share our assets with you, you set a link to icons8.com wherever you use them. You get top-notch graphics for free, we get some exposure. A win-win! Below you find example with the most frequent use cases.

https://intercom.help/icons8-7fb7577e8170/en/articles/5534926-universal-multimedia-license-agreement-for-icons8

**Comment by sayyid5416** [comment]
@huynhsontung Is there any update regarding this or the project is on halt?

**Reply by huynhsontung** [reply]
@sayyid5416 I am taking a break from coding so expect to see low activities on this project for the next while. I am still working on major features in the background, which you can track through my pull requests. Icon update is not a critical change so it's been deferred until closer to v1 release.

**Comment by huynhsontung** [comment]
Below is the prospective main icon. File icons (such as audio, video, and playlist icons) are not yet decided, and I would love to have some help on.
<img width="250" alt="logo-redesign-artboard11" src="https://github.com/huynhsontung/Screenbox/assets/31434093/49d02811-f9e9-41d2-9143-83c9d4a75629">

**Reply by CaptechOmar** [reply]
Why is the current icon different from this one?

**Reply by huynhsontung** [reply]
After some discussion on Discord, we agreed that even though the icon looks nice. It doesn't fit the Fluent Design style on Windows. The current icon was used instead.

**Comment by huynhsontung** [comment]
The new icon will be implemented in #443
